src/renderer.js

```diff
@@ -41,8 +41,26 @@ setButton.addEventListener("click", () => {
   newTaskItem.appendChild(taskDescription);
   newTaskItem.appendChild(removeText);
 
+  const currentDay = new Date();
```

Comment:
This is very annoying and repetitive and I will fix it in the feature by bringing these variables into global ones so they don't get initialised all the time but for now I am trying to make everything work